pyinstaller/trader_bin.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Script for building the AEA responsible for running the trader."""
+import os
+import sys
+from pathlib import Path
+import aea.configurations.validation as validation_module
+
+# patch for the _CUR_DIR value
+validation_module._CUR_DIR = Path(sys._MEIPASS) / validation_module._CUR_DIR
```

Comment:
Is this only valid for Mac ?

Reply:
Could you update the comment to explain why we need this patch?

Reply:
no, its for all, just a pyinstaller thing

Reply:
added a comment